src/WrapperApp/components/Simulation/SimulationCardGrid.tsx

```diff
@@ -83,19 +84,25 @@ export function SimulationCardGrid({
 	handleShowInputFiles,
 	...other
 }: SimulationCardGridProps) {
-	let gridContainerProps: GridProps = { container: true };
-	let gridItemProps: GridProps = { item: true };
+	let gridContainerProps: GridProps = { container: true, width: '100%' };
+	let gridItemProps: GridProps = {}; // default grid item props
 
 	if (validGriLayout(layout)) {
 		gridContainerProps = {
 			...gridContainerProps,
 			...stylesByLayout[layout].gridContainerProps
 		};
-		gridItemProps = { ...gridItemProps, ...stylesByLayout[layout].gridItemProps };
+
+		gridItemProps = {
+			...gridItemProps,
+			...stylesByLayout[layout].gridItemProps
+		};
 	} else {
 		console.warn(`Unknown layout: ${layout}`);
 	}
 
+	console.log(sx);
```

Comment:
Looks good, but I get a lot of such printouts:
![image](https://github.com/user-attachments/assets/89bd674b-885c-47ee-837a-0803e7c1f740)

do we need this `console.log()` ?